libs/libcoap/patches/0001-manpage.patch

```diff
@@ -0,0 +1,21 @@
+--- a/examples/Makefile.am
++++ b/examples/Makefile.am
+@@ -23,18 +23,6 @@
+ coap_rd_SOURCES = coap-rd.c
+ coap_rd_LDADD = @top_builddir@/.libs/libcoap-$(LIBCOAP_API_VERSION).la
+ 
+-# building the manpages
```

Comment:
Yeah, --disable-documentation doesn't eliminate man5 generation. Let's handle it by patch for a while, and I'll try to figure out with it and make patch for upstream source soon.